server/s3_put_object_action.cc

```diff
@@ -898,6 +901,13 @@ void S3PutObjectAction::send_response_to_s3_client() {
 
     request->set_out_header_value("ETag", e_tag);
 
+    std::string versioning_status =
+        bucket_metadata->get_bucket_versioning_status();
+    if ("Unversioned" != versioning_status) {
+      std::string ver_id = "\"" + object_metadata->get_obj_version_id() + "\"";
+      request->set_out_header_value("VersionId", ver_id);
```

Comment:
```suggestion
      request->set_out_header_value("x-amz-version-id ", ver_id);
```

https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/768639283/PutObject+LLD#Response-Syntax